### PR TITLE
created new ai webinar

### LINKED ISSUE #4087: AI webinar takeover - Build
https://docs.google.com/document/d/1P8BC483zp5bYS5SuKimBHSIBlLhyEL9O_AcH3-pAjVc/edit#

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created a duplicate of the original AI webinar
- TODO - add fing to rotation of takeovers when ready

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Load the homepage until the take over appears, look at S,M,L and compare to the [copy doc](https://docs.google.com/document/d/1P8BC483zp5bYS5SuKimBHSIBlLhyEL9O_AcH3-pAjVc/edit#)

## Issue / Card

Fixes #4087

### CODE REVIEW

File: templates/takeovers/_ai_webinar-2018-10-01.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      <div class="p-takeover__image-container u-align-text--center">
+        <img class="dotted-circle" src="{{ ASSET_SERVER_URL }}9f3f2b68-dotted-circle.svg" alt="">
+        <img class="ubuntu-logo" src="{{ ASSET_SERVER_URL }}54e24b4c-ubuntu-logo.svg" alt="">
+        <img class="desktop-picto" src="{{ ASSET_SERVER_URL }}5e8d3f1a-desktop-icon.svg" alt="">
+        <img class="iot-picto" src="{{ ASSET_SERVER_URL }}c2715a27-iot-icon.svg" alt="">
+        <img class="cloud-picto" src="{{ ASSET_SERVER_URL }}82c26ccc-cloud-icon.svg" alt="">
+        <img class="server-picto" src="{{ ASSET_SERVER_URL }}fd20f946-server-icon.svg" alt="">
+      </div>
+      <div class="u-hide--medium u-hide--large">
+        <a href="https://www.brighttalk.com/webcast/6793/336267" class="p-button--positive u-no-margin--bottom">Watch on-demand</a>
```
Comment (anthonydillon) [inline templates/takeovers/_ai_webinar-2018-10-01.html]: Should be `Register for the webinar`

Comment (pmahnke) [reply templates/takeovers/_ai_webinar-2018-10-01.html]: @anthonydillon 	thanks ... good catch

**Comment by anthonydillon** [comment]
@pmahnke you have conflicts.

File: templates/takeovers/_ai_webinar-2018-10-01.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h1 class="p-takeover__title">How to build and deploy your first AI/ML model on
         <img class="server-picto" src="{{ ASSET_SERVER_URL }}fd20f946-server-icon.svg" alt="">
       </div>
       <div class="u-hide--medium u-hide--large">
-        <a href="https://www.brighttalk.com/webcast/6793/336267" class="p-button--positive u-no-margin--bottom">Watch on-demand</a>
+        <a href="https://www.brighttalk.com/webcast/6793/336267" class="p-button--positive u-no-margin--bottom">Register for the webinar</a>
```
Comment (caldav) [inline templates/takeovers/_ai_webinar-2018-10-01.html]: Please use the landing page for the URL: `/engage/deploy-ai-ml-model?utm_medium=takeover&utm_campaign=FY19_Cloud_K8_WBN_AIWebinar2`

**Pull request merged**